### DISCUSSION TITLE
Actividad 3 - Grupo 8

### DISCUSSION THREAD

**Discussion by tomaszbk** [frlp-utn-ingsoft/catedra#134] [Show and tell]
https://github.com/Salani1003/vetsoft

**Comment by ivanduva** [comment]
Buenas!
La verdad que la actividad está en general bastante bien. Van a tener que corregir un par de cosas porque se rompe la app en un caso y en otro no resuelve totalmente el issue, pero más allá de eso vemos buenos tests y una buena review en los PRs.
Los problemas son los siguientes:

- El form de producto tira 500 al ingresar un caracter no numérico o un float con coma.
- La fecha de nacimiento de la mascota permite agregar fechas futuras

También les dejamos algunos comentarios para que mejoren en las próximas entregas:

- En general los commits están bien, pero si van a estandarizar usen todos el mismo formato. Vimos algunos mensajes que empiezan con "feat:", otros con "Feature /". También vimos algunos commits muy atómicos y otros nada atómicos. Traten de ponerse de acuerdo en ese sentido.
- Lo mismo en el caso de los nombres de los branches.
- En los tests e2e, a veces repiten algunos expect que ya se han hecho antes. Por ejemplo, no es necesario preguntar si el form está visible en cada test si antes ya lo hicieron en otro.

Buen trabajo! Esperamos esos cambios y ya con eso tendrían aprobada la actividad.
Saludos!

**Comment by tomaszbk** [comment]
Buenas noches! ya arreglamos los errores del form de producto y de la fecha de nacimiento de la mascota, en 2 nuevos PR respectivos. Saludos!

**Comment by ivanduva** [comment]
Buenas,
Lo de la fecha está muy bien, el test incluido.
En cuanto al precio del producto, ahora el form de editar el producto no está validando el precio y se permite modificar productos con precio 0 o menor.
Arreglen eso y agreguen el test para asegurar. Lo debe hacer la persona que está encargada de ese issue. Y quien vaya a hacer la review, que pruebe bien porque estos problemas se deberían atajar con una buena review.

Saludos

**Reply by tomaszbk** [reply]
Arreglado issue del producto por Esteban 👍

**Comment by ivanduva** [comment]
Genial, queda aprobada la actividad.
Saludos
